### PR TITLE
Implements AD/LDAP Import schedule

### PR DESCRIPTION
**Pull request by wivern**
Closes OHDSI/WebAPI#640

### CODE REVIEW

**Comment by pavgra** [comment]
+ address column widths please so that they don't jump

File: js/pages/configuration/users-import/browser.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		Const,
+		Utils,
+) {
+		class UserImportBrowser extends AutoBind(Component) {
+
+			constructor(params) {
+				super();
+				this.config = config;
+				this.isAuthenticated = authApi.isAuthenticated;
+				this.canImport = ko.pureComputed(() => this.isAuthenticated() && authApi.isPermittedImportUsers());
```
Comment (pavgra) [inline js/pages/configuration/users-import/browser.js]: `isAuthenticated()` is redundant - there is such check inside `AuthAPI.isPermitted` used by `isPermittedImportUsers`

File: js/pages/configuration/users-import/browser.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        </button>
+        <button data-bind="
+            css: classes({ element: 'new-entity-btn', extra: 'btn btn-primary btn-sm' }),
+            text: 'New Scheduled Import',
+            click: $component.newScheduledImport
+        "></button>
+    </div>
+</div>
+
+<div data-bind="if: config.userAuthenticationEnabled && canImport, css: classes('grid')">
```
Comment (pavgra) [inline js/pages/configuration/users-import/browser.html]: `userAuthenticationEnabled` is redundant. see comment in controller

File: js/pages/configuration/users-import/components/job-history.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+<script type="text/html" id="exit-message-template">
+<div data-bind="text: message"></div>
+</script>
+
+<atlas-modal params="
+    showModal: messageModal,
+    title: 'Job Exit Message',
+    template: 'exit-message-template',
+    data: { message },
+">
```
Comment (pavgra) [inline js/pages/configuration/users-import/components/job-history.html]: Please put markup inside the atlas modal tag, no need for the separate template:

```
<atlas-modal params="...">
  <div data-bind="text: message"></div>
</atlas-modal>
```

File: js/pages/configuration/users-import/services/PermissionService.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+define([
+	'services/AuthAPI',
+], function (
+	AuthAPI,
+) {
+
+	function isPermittedCreate() {
```
Comment (pavgra) [inline js/pages/configuration/users-import/services/PermissionService.js]: Where is it used?

File: js/pages/configuration/users-import/services/PermissionService.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+		return AuthAPI.isPermitted('user:import:job:post');
+	}
+
+	function isPermittedList() {
```
Comment (pavgra) [inline js/pages/configuration/users-import/services/PermissionService.js]: Where is it used?

File: js/pages/configuration/users-import/services/PermissionService.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+		return AuthAPI.isPermitted('user:import:job:get');
+	}
+
+	function isPermittedView(id) {
```
Comment (pavgra) [inline js/pages/configuration/users-import/services/PermissionService.js]: Where is it used?

File: js/assets/bootstrap-datetimepicker.min.js [hunk 7]
```diff
@@ -0,0 +1,2 @@
... parameter");return d.keepOpen=a,l},l.focusOnShow=function(a){if(0===arguments.length)return d.focusOnShow;if("boolean"!=typeof a)throw new TypeError("focusOnShow() expects a boolean parameter");return d.focusOnShow=a,l},l.inline=function(a){if(0===arguments.length)return d.inline;if("boolean"!=typeof a)throw new TypeError("inline() expects a boolean parameter");return d.inline=a,l},l.clear=function(){return ca(),l},l.keyBinds=function(a){return 0===arguments.length?d.keyBinds:(d.keyBinds=a,l)},l.getMoment=function(a){return y(a)},l.debug=function(a){if("boolean"!=typeof a)throw new TypeError("debug() expects a boolean parameter");return d.debug=a,l},l.allowInputToggle=function(a){if(0===arguments.length)return d.allowInputToggle;if("boolean"!=typeof a)throw new TypeError("allowInputToggle() expects a boolean parameter");return d.allowInputToggle=a,l},l.showClose=function(a){if(0===arguments.length)return d.showClose;if("boolean"!=typeof a)throw new TypeError("showClose() expects a boolean parameter");return d.showClose=a,l},l.keepInvalid=function(a){if(0===arguments.length)return d.keepInvalid;if("boolean"!=typeof a)throw new TypeError("keepInvalid() expects a boolean parameter");
```
[the remark this answers is not in the document]
Comment (wivern) [reply js/assets/bootstrap-datetimepicker.min.js]: I didn't find it in any CDNs

File: js/pages/configuration/users-import/services/PermissionService.js (hunk 6 above)
Comment (wivern) [reply js/pages/configuration/users-import/services/PermissionService.js]: it used in isPermittedEdit

**Pull request merged**